javascript/javascript1/week1/startupNames.js

```diff
@@ -0,0 +1,36 @@
+//4. Ez Namey (Startup name generator) Optional
+
+
+//First array Names:
+let firstWords = [
+    "Gorgeous",
+    "Creative",
+    "Extraordinary",
+    "Cool",
+    "Smart",
+    "Hero",
+    "Ultimate",
+    "NextGen",
+    "Super",
+    "Wisdom",
+  ];
+  
+  //Second array Names:
+  let secondWords = [
+    "Vision",
+    "Thought",
+    "Concepts",
+    "Technologies",
+    "Systems",
+    "Preparation",
+    "Inventions",
+    "Process",
+    "Tools",
+    "Product",
+  ];
+  const randomNumber = Math.floor(Math.random() * 10);
```

Comment:
Using a single random number for both first and second words would give you always the same pairs. If you add another variable for a second random number that you can use at `secondWords`, you would get much bigger variety of combinations.

Reply:
I do not understand about comment, how to do that?